Fallout2/Fallout1in2/ddraw.ini

```diff
@@ -367,7 +367,8 @@ ElevatorsFile=config\Elevators.ini
 
 ;Uncomment and set a comma delimited list of numbers to use a custom xp table.
 ;Player's level is capped once the highest specified level is reached
-XPTable=1000,3000,6000,10000,15000,21000,28000,36000,45000,55000,66000,78000,91000,105000,120000,136000,153000,171000,190000,210000
+;Max Level +1, because the last level will not be reached until changes in the code, so want to reach max Level 21, put in a 22nd (Start is Level 1 with 0xp and not entered)
+XPTable=1000,3000,6000,10000,15000,21000,28000,36000,45000,55000,66000,78000,91000,105000,120000,136000,153000,171000,190000,210000,231000

```

Comment:
Level 21 works, this change is unnecessary. https://i.imgur.com/Y9Tusb0.png

Reply:
Okay, guess many people are simply around with older sfall. reverted